src/Apps/Search/SearchApp.tsx

```diff
@@ -24,6 +25,48 @@ export interface Props {
   context_page: Schema.PageName.SearchPage,
 })
 export class SearchApp extends React.Component<Props> {
+  renderResults(count: number, artworkCount: number) {
```

Comment:
Adjusted the app to display the zero state (w/o rendering the '0 results for andy' header, or any tabs).